.github/workflows/test.yml

```diff
@@ -84,77 +84,6 @@ jobs:
           path: json.log
           retention-days: 30
 
-  test_coverage:
```

Comment:
yaml for > 5 lines is not the future -- please double check I scoped the delete properly